solv/tdnfpool.c

```diff
@@ -67,7 +67,8 @@ uint32_t
 SolvInitSack(
     PSolvSack *ppSack,
     const char* pszCacheDir,
-    const char* pszRootDir
+    const char* pszRootDir,
+    int nSkipInstalledRpms
```

Comment:
shall we rename nSkipInstalledRpms to nInstallAllDeps as it looks little confusing? What are your thoughts?

I understand concept is same, but it feels easy to read and understand code with this name. I am open for any new suggestions as well.

Reply:
Sure, I can do that.